include/motor-control/core/motor_interrupt_handler.hpp

```diff
@@ -136,6 +129,7 @@ class MotorInterruptHandler {
     auto homing_stopped() -> bool {
         if (limit_switch_triggered()) {
             finish_current_move(AckMessageId::stopped_by_condition);
+            position_tracker = 0;
```

Comment:
I have a question about the reset function: does queue.reset() reset the entire queue, or just pop the current move off of it? If it's just the last move, could we just use the reset() in this file and start the next move?

Reply:
Looks like it [removes all the messages](https://www.freertos.org/a00018.html#xQueueReset)

Reply:
gotcha